client/api/disk/v1beta3/api.proto

```diff
@@ -41,69 +41,71 @@ message DiskLocation {
 }
 
 message ListDiskLocationsResponse {
-    // Map of disk device IDs and <adapter, bus, target, lun ID> associated with each disk device
-    map <string, DiskLocation> disk_locations = 1;
+    // Map of disk number and <adapter, bus, target, lun ID> associated with each disk device.
```

Comment:
could we define enum here so that it is more clear what are the keys of the map?

Reply:
this is the same as https://github.com/kubernetes-csi/csi-proxy/pull/140#discussion_r642623506